labs/lab6/lab6.adoc

```diff
@@ -29,56 +37,311 @@ Password: INSTRUCTOR WILL PROVIDE
+
+Navigate to the OpenShift Web Console and login as the _<student_id>-admin_ user.
+
+link:https://master-<student_id>.labs.sysdeseng.com:8443[https://master-<student_id>.labs.sysdeseng.com:8443]
+
+After logging in, locate "My Projects" on the righthand side and select **View All** to view all projects the user has access to. Select **openshift-metrics**.
+
+To deploy the APB to the project, select **Add to Project** and then **Browse Catalog** from the link on the top righthand side of the page.
+
+Locate and select _AlertManager Ansible Tower Bridge_ from the catalog.
+
+image::images/openshift-metrics-catalog.png[]
+
+A dialog will appear to walk through the process of deploying the application. Click **Next** at the bottom right to process to the next dialog.
+
+Several fields are defined on the Configuration page. The majority of these fields can be left to their default values. The following fields must be specified:
+
+Ansible Tower Hostname: **tower-<student_id>.labs.sysdeseng.com** +
+User with permissions on Tower: **<student_id>** +
+Password for user with permission on Tower: **<provided_password>** +
+Retype password for user with permission on Tower: **<provided_password>**
+
+Click **Create** to provision the service which will launch the Ansible Playbook Bundle.
 
-If the web session has not been retained from a prior lab, login once again.
+The APB will create a Secret containing credentials, a BuildConfig to build a new image containing the application, a Service and a DeploymentConfig to deploy the application after the image has been built.
 
-After logging in, navigate to the **TEMPLATES** page and locate the **2-Scaleup_OpenShift_on_AWS** workflow job template. Click the **RocketShip** icon to start the job.
+Click the **Overview** tab to view the status of the image build and deployment.
 
-image::images/ansible-tower-scaleup-template.png[]
+image::images/alertmanager-tower-bridge-app-overview.png[]
 
-The workflow first creates a new AWS instance and once the instance has been created, the scaleup Ansible playbook will be executed to expand the cluster. Finally, a third job template performs final actions to finalize the cluster scaleup.
+A running pod similar to the image above indicates the application deployment was successful. 
+
+== Scaling the Cluster
+
+With the components in place to integrate AlertManager with Ansible Tower, lets turn our attention into how the cluster can be dynamically scaled. Alerts are sent to AlertManager based on rules that have been defined within Prometheus. The rules and their current status can be viewed from the Prometheus User Interface.
+
+Navigate to the Prometheus user interface and login using the "<student_id>-admin" user account
+
+link:https://prometheus-openshift-metrics.apps-<student_id>.labs.sysdeseng.com[https://prometheus-openshift-metrics.apps-<student_id>.labs.sysdeseng.com]
```

Comment:
They will have to allow permissions the first time they hit this page.

Reply:
Addressed in updated PR